force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -270,18 +343,19 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<Book> generateBookList(Book myBook1) {
         // Create a list of books
-
+        List<book> myBooks = new List<Book>();
         // Update the title and author of myBook1
-
+        Book book1 = new Book(myBook1.title,myBook1.author);
```

Comment:
Try to add these values to the book using these literal strings to replace what was there already
 "Harry Potter and the Chamber of Secrets", author = "J.K. Rowling"